src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -15,28 +15,32 @@ Watch this short video (approx. 2:15 minutes) to learn how to:
 
 To use distributed tracing, you need at least XCFramework agent version 7.3.0. We recommend you use the most recent agent.
 
-<Callout variant="tip">
-Infinite Tracing is not yet available for mobile monitoring.
-</Callout>
-
-## How to set up distributed tracing [#setup]
+## How to set up distributed tracing [#set-up-dt]
 
 For mobile agents that support this feature, it’s enabled by default. 
 
 If you prefer to turn off distributed tracing, see [iOS feature flag](/docs/mobile-monitoring/new-relic-mobile-ios/api-guides/ios-agent-configuration-feature-flags/#dt).
 
+## Infinite Tracing [#infinite-tracing]
+
+If your mobile apps have some downstream services that use the advanced type of distributed tracing called Infinite Tracing, you should enable Infinite Tracing for your mobile apps. This ensures that your root span (the initiating mobile app) is included in the rest of the spans tracked by Infinite Tracing.
+
+Infinite Tracing uses a trace observer in the cloud to collect all your trace spans. The trace observer uses [tail-based sampling](https://docs.newrelic.com/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#tail-based) to sort through your traces. This means it waits until all your spans arrive before assembling them into traces. Then, it looks through all your traces to find the important ones. 
+
+To set up Infinite Tracing, go to our [Infinite Tracing](/docs/distributed-tracing/infinite-tracing/set-trace-observer) documentation for steps to create a trace observer and to select which mobile apps (data sources) should send trace spans to the Infinite Tracing trace observer.
+
 ## Find data [#find-data]
 
 Mobile spans appear in any [New Relic distributed tracing UI](/docs/understand-dependencies/distributed-tracing/ui-data/additional-distributed-tracing-features-new-relic-one/) where those spans are part of a trace. 
 
 Here are some tips for finding and querying data:
 
 * You can find end-user-originating traces in any [New Relic One distributed tracing UI](/docs/understand-dependencies/distributed-tracing/ui-data/additional-distributed-tracing-features-new-relic-one).
-* In the distributed tracing UI, end-user spans are indicated with the ![New Relic distributed tracing browser span icon](./images/mobile-icon.png "distributed-tracing-browser-span-icon.png") icon.
+* In the distributed tracing UI, end-user spans are indicated with the ![New Relic distributed tracing mobile span icon](./images/mobile-icon.png "New Relic distributed tracing mobile span icon") icon.
 * To see a span's attributes, select [a span in the UI](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data#span-details).
 * Spans are reported as [Span data](/attribute-dictionary/?event=Span), and can be [queried in New Relic](/docs/using-new-relic/data/understand-data/query-new-relic-data).
 * Query tips:
-  * Query by browser app name by setting `mobileApp.name` to the browser app name.
```

Comment:
oh whoops, that was likely my bad.

Reply:
@zuluecho9, I thought I had caught those stray browser references!